modules/nodes-cluster-resource-levels-command.adoc

```diff
@@ -44,44 +44,51 @@ spec:
 
 .Procedure
 
-To use the tool on the command line: 
+To use the tool on the command line:
 
-. Run the following command:
+. From the terminal, run:
++
+----
+podman login registry.redhat.io
+----
++
+
+. Run:
 +
-[source,terminal]
 ----
-$ ./cluster-capacity --kubeconfig <path-to-kubeconfig> \ <1>
-    --podspec <path-to-pod-spec> <2>
+podman pull registry.redhat.io/openshift4/ose-cluster-capacity
 ----
-<1> Specify the path to your Kubernetes configuration file.
-<2> Specify the path to the sample `Pod` spec file
 +
-You can also add the `--verbose` option to output a detailed description of how
-many pods can be scheduled on each node in the cluster:
+
+. Run:
 +
 [source,terminal]
 ----
-$ ./cluster-capacity --kubeconfig <path-to-kubeconfig> \
-    --podspec <path-to-pod-spec> --verbose
+$ podman run -v $HOME/.kube:/kube:Z -v $(pwd):/cc:Z  ose-cluster-capacity
```

Comment:
This is a question for the SME/QE, but I don't think you can wrap this command to additional lines without a `\` continuation character at the end of the first two lines

Reply:
@bergerhoffer True, we'd need to include the \ if we should be able to cut and paste directly from the command section

Reply:
Done